tests/AutoReview/CiConfigurationTest.php

```diff
@@ -275,11 +275,25 @@ private function getPhpVersionsUsedForBuildingOfficialImages(): array
      */
     private function getPhpVersionsUsedForBuildingLocalImages(): array
     {
-        $yaml = Yaml::parse(file_get_contents(__DIR__.'/../../.github/workflows/docker.yml'));
+        $yaml = self::parseYamlFromFile(__DIR__.'/../../.github/workflows/docker.yml');
```

Comment:
Why not:

```
$yaml = Yaml::parseFile(__DIR__.'/../../.github/workflows/docker.yml');
```

It throws `ParseException` when file is not readable or contains invalid YAML. Should be enough, without custom method.

Reply:
pleae go ahead 👍🏻

Reply:
#8380